### PR TITLE
Add bamplot

### PR DESCRIPTION
**Pull request by JenkoB**
Still need to test with gff file and bed file.

### CODE REVIEW

File: resolwe_bio/processes/plots/bamplot.yml [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+      required: false
+      description: >
+         Enter .gff file.
+    - name: i_region
+      label: Region string
+      type: basic:string
+      required: false
+      description: >
+         Enter genomic region e.g. chr1:+:1-1000.
+    - name: b
```
Comment (dblenkus) [inline resolwe_bio/processes/plots/bamplot.yml]: Use more verbose attribute names. It is much easier to understand code with `bam`, `color`, `sense`..., than just `b`, `c`, `s`...

File: resolwe_bio/processes/plots/bamplot.yml [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+        bamplot -g {{g}} -i {{i_region}} -b {% for bam in b %}{{bam.bam.file}} {% endfor %}-c {{c}} -e {{e}} -s {{s}} -y {{y}} {% if r %}-r{% endif %} {% if n %}-n {{n}}{% endif %} -p {{p}} {% if scale %}-scale {{scale}}{% endif %} {% if bed %}--bed {{bed.BED.file}}{% endif %} {% if stretch_input %}--stretch-input {{stretch_input}}{% endif %} {% if multi_page %}--multi-page{% endif %} {% if verbose %}--verbose{% endif %} -o output_plot -t {{t}}
+
+        re-checkrc "bamplot failed."
+
+        {% if multi_page %}
+          {% if t != "output" %}
+            re-save-file plot output_plot/{{t}}_plots_NA.pdf
+
+          {% else %}
+            m mv output_plot/{{t}}_plots_NA.pdf output_plot/{{i_region}}_plots_NA.pdf
```
Comment (dblenkus) [inline resolwe_bio/processes/plots/bamplot.yml]: `m mv` -> `mv`?

File: resolwe_bio/processes/import_data/annotation.yml [hunk 3]
```diff
@@ -65,14 +65,15 @@
   category: upload
   persistence: RAW
   description: >
-    Upload annotation in GTF format.
+    The GTF format is identical to GFF version 2 so this process
```
Comment (tjanez) [inline resolwe_bio/processes/import_data/annotation.yml]: Wouldn't it be more descriptive to also preserve the original description (i.e. "Upload annotation in GTF format.") and add the new text as an addition?

File: resolwe_bio/tests/files/ALL_SE_sorted1.bed [hunk 4]
```diff
@@ -0,0 +1,1859 @@
+chr1	9996	40602	1_group4_wnt_1559_lociStitched
```
Comment (tjanez) [inline resolwe_bio/tests/files/ALL_SE_sorted1.bed]: Would it make sense to shorten this test bed file? Does it have any influence on the time needed for the tests to run?

Also, existing files in `resolwe_bio/tests/files` usually use lower case for names and the prefix says something where this is used. Could you improve the name?

File: resolwe_bio/tests/files/HG19_FIGURE1.gff [hunk 5]
```diff
@@ -0,0 +1,16 @@
+chr17	ARL4D	ARL4D	41468594	41566948		+		ARL4D
```
Comment (tjanez) [inline resolwe_bio/tests/files/HG19_FIGURE1.gff]: Similar as above, existing files in `resolwe_bio/tests/files` usually use lower case for names and the prefix says something where this is used. Could you improve the name?

File: resolwe_bio/processes/import_data/annotation.yml (hunk 3 above)
Comment (JenkoB) [inline resolwe_bio/processes/import_data/annotation.yml]: Resolved.

File: resolwe_bio/tests/files/ALL_SE_sorted1.bed (hunk 4 above)
Comment (JenkoB) [inline resolwe_bio/tests/files/ALL_SE_sorted1.bed]: The name is changed and the file is shortened.

File: resolwe_bio/tests/files/HG19_FIGURE1.gff (hunk 5 above)
Comment (JenkoB) [inline resolwe_bio/tests/files/HG19_FIGURE1.gff]: Corrected.

File: resolwe_bio/processes/import_data/annotation.yml [hunk 6]
```diff
@@ -65,14 +65,16 @@
   category: upload
   persistence: RAW
   description: >
-    Upload annotation in GTF format.
+    Upload annotation in HTF format. The GTF format is identical to GFF
```
Comment (dblenkus) [inline resolwe_bio/processes/import_data/annotation.yml]: `HTF` -> `GTF`

**Comment by tjanez** [comment]
@JenkoB, looks good!

**Comment by mstajdohar** [comment]
Fix tests and docs.

**Pull request merged**